### ISSUE TITLE
"minikube start --driver=docker" not working as expected 

### ISSUE THREAD

**Issue by prudhvinag06** [kubernetes/minikube#13588]
### What Happened?

![Screenshot from 2022-02-09 15-33-38](https://user-images.githubusercontent.com/88816987/153175769-3fade431-dc7c-4bce-a09e-e19d42bc612f.png)
GUEST_START error
Any idea how to fix this issue

### Attach the log file

[logs.txt](https://github.com/kubernetes/minikube/files/8031590/logs.txt)

### Operating System

Ubuntu

### Driver

Docker

**Comment by medyagh** [comment]
@prudhvinag06 it seems like ur kube config folder is messed up, do mind sharing

`ls -lah ~/.kube`
 ~/.kube/config is supposed to be a file not a folder ! u might wanna backup and delete that folder and then try again

/triage needs-information
/kind support

**Comment by kimba74** [comment]
I had a similar problem after upgrading to v1.25.1. After some time I realized that for whatever reason the docker network the container was configured to use got deleted (or wasn't existing). After re-creating and connecting a docker network to the minikube container it started working for me again.

The odd thing was when I created three different minikubes via profiles in a row, after I shut them all down only the last one created could be started again the other ones, despite having different names, had their docker network erased.
